agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/localstorage/LocalFileWriter.java

```diff
@@ -118,12 +118,20 @@ void writeToDisk(String instrumentationKey, List<ByteBuffer> buffers) {
     operationLogger.recordSuccess();
   }
 
-  private static void write(File file, List<ByteBuffer> buffers, String instrumentationKey)
+  private static void write(File file, String connectionString, List<ByteBuffer> buffers)
       throws IOException {
-    try (FileChannel channel = new FileOutputStream(file).getChannel()) {
-      channel.write(ByteBuffer.wrap(instrumentationKey.getBytes(UTF_8)));
+
+    try (FileOutputStream fileOut = new FileOutputStream(file);
+        DataOutputStream dataOut = new DataOutputStream(fileOut)) {
+      dataOut.writeInt(1); // version
```

Comment:
not sure i understand the purpose of writing verison though.

Reply:
so that in the future (if we have version 2), we have a better way of checking if it's an old format that we don't support